src/components/structures/MatrixChat.js

```diff
@@ -607,6 +607,7 @@ module.exports = React.createClass({
     // @param {boolean=} roomInfo.show_settings Makes RoomView show the room settings dialog.
     // @param {string=} roomInfo.event_id ID of the event in this room to show: this will cause a switch to the
     //                                    context of that particular event.
+    // @param {boolean=} roomInfo.highlighted If true, add event_id to the hash of the URL
```

Comment:
OK, but I feel that someone is going to get confused when the documentation for this function doesn't match what the function does.